e2e/vaultsecrets/input/secrets.nomad

```diff
@@ -0,0 +1,55 @@
+job "secrets" {
+  datacenters = ["dc1", "dc2"]
+
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
```

Comment:
any particular reason to limit this to linux?

Reply:
Right now all E2E jobs that run on Docker have to run on Linux because we don't have a Windows 2019 target. That's coming but not yet.